packs/data/other-effects.db/effect-dazzled-until-end-of-your-next-turn.json

```
@@ -0,0 +1,47 @@
+{
+    "_id": "uwYGEeqvt8pwjhXa",
+    "img": "systems/pf2e/icons/effects/critical-effect.webp",
+    "name": "Effect: Dazzled until end of your next turn",
+    "system": {
+        "badge": null,
+        "description": {
+            "value": "<p><strong>Effect applied by @UUID[Compendium.pf2e.criticaldeck.Critical Fumble Deck #51]{Critical Fumble Deck #51}, @UUID[Compendium.pf2e.criticaldeck.Critical Fumble Deck #52]{Critical Fumble Deck #52}</strong></p>\n<p data-pm-slice=\"1 1 [&quot;section&quot;,{&quot;_preserve&quot;:{&quot;class&quot;:&quot;fumble-deck&quot;}},&quot;blockquote&quot;,{&quot;_preserve&quot;:{}}]\">You are <a class=\"content-link\" draggable=\"true\" data-uuid=\"Compendium.pf2e.conditionitems.TkIyaNPgTZFBCCuh\" data-id=\"TkIyaNPgTZFBCCuh\" data-type=\"Item\" data-pack=\"pf2e.conditionitems\" data-tooltip=\"Condition Item\" data-args=\"\">Dazzled</a> until the end of your next turn.</p>"
+        },
+        "duration": {
+            "expiry": "turn-end",
+            "sustained": false,
+            "unit": "rounds",
+            "value": 1
+        },
+        "level": {
+            "value": 1
+        },
+        "rules": [
+            {
+                "key": "GrantItem",
+                "onDeleteActions": {
+                    "grantee": "restrict"
+                },
+                "uuid": "Compendium.pf2e.conditionitems.Deafened"
+            }
+        ],
+        "source": {
+            "value": ""
```

```suggestion
            "value": ""
```

Forgot to flag adding this on the other PR but sources needed on these. We should surface that field on effects somewhere.

---

Added source.